src/Command/Push/PushArtifactCommand.php

```diff
@@ -160,8 +160,8 @@ private function determineDestinationGitUrls(?string $applicationUuid): array
         if ($envVar = getenv('ACLI_PUSH_ARTIFACT_DESTINATION_GIT_URLS')) {
             return explode(',', $envVar);
         }
-        if ($this->datastoreAcli->get('push.artifact.destination-git-urls')) {
-            return $this->datastoreAcli->get('push.artifact.destination-git-urls');
+        if ($this->datastoreAcli->get('push.artifact.destination_git_urls')) {
```

Comment:
change from `destination-git-urls` to `destination_git_urls` is doing because for some reason (not sure why) if in https://github.com/acquia/cli/pull/1789/files#diff-e28b34b7f9414da3580d7e2274fd17c002beac308f6ab89aa7566f258198cd3aR28 use without underscrore that doesn't work but with underscore works.

So to me looks like somewhere this `-` is getting replaced with `_`

Reply:
We should be using underscores anyway since that's the convention (see `cloud_app_uuid`)